base/main.cpp

```diff
@@ -427,6 +427,10 @@ extern "C" int scummvm_main(int argc, const char * const argv[]) {
 		ConfMan.loadDefaultConfigFile();
 	}
 
+	if(settings.contains("savepath")){
+		ConfMan.set("savepath", settings["savepath"], Common::ConfigManager::kSessionDomain); 
+	}
+
```

Comment:
I am wondering if it would not be better to have this toward the end of the `processSettings()` function. Currently this is where all the settings are copied to the transient domain and it seems to me it would make sense to have the copy to the session domain in the same place.